docs/requirements.txt

```diff
@@ -1,4 +1,3 @@
-pymdown-extensions==8.0.1
-markdown-include==0.6.0
-mkdocs-material==7.2.4
-jinja2==3.0.3
```

Comment:
just to confirm that this package is no longer needed?

Reply:
It no longer needs to be pinned